### ISSUE TITLE
Script rework

### ISSUE THREAD

**Issue by HenryDane** [HenryDane/EunioaWebsite#8]
Scripts need to be reworked, passwords changed, databases rebuilt. I will handle this part so no worries.

**Comment by artillect** [comment]
We should remove as many scripts as possible, I believe it may be impacting website performance/loading speed as referred to in #7

**Comment by HenryDane** [comment]
I disagree -- currently there are no scripts uploaded to the server. The links to them are disabled.

**Comment by artillect** [comment]
Oh, I didn't realize they weren't even enabled. How is the "contact us"
form supposed to work then?

**Comment by rey-grschel** [comment]
Ok github break my email formatting why don’t ya

**Comment by HenryDane** [comment]
It isn't.
